app/controllers/application_controller.rb

```diff
@@ -18,6 +18,9 @@ def set_locale
       # { :locale => I18n.locale }
   # end
 
+  rescue_from(ActionController::ParameterMissing) do |e|
+    render text: "Request is missing param '#{e.param}'", status: :bad_request
```

Comment:
I was wondering to rescue differently for API vs web, but wanted to keep this simpler, and change afterwards if necessary.
The current behaviour right now is a render of text, even on API, so.